zcutil/distclean.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# Copyright (c) 2020 The Zcash developers
+
+zcutil/clean.sh
+
+rm -rf depends/*-*-*
+rm -rf depends/built
+rm -rf depends/sources
+rm -rf target
+
+rm -f .cargo/config
+rm -f .cargo/.configured-for-offline
```

Comment:
- These are generated by `src/Makefile.am`, so should be removed by `zcutil/clean.sh`
- Missing `.cargo/.configured-for-online`